### PR TITLE
fix state-auth

### PR DESCRIPTION
**Pull request by sFLASHs**

### CODE REVIEW

File: client/src/containers/routing/index.tsx [hunk 1]
```diff
@@ -47,7 +48,7 @@ const Routing: React.FC<IRoutingProps> = () => {
 			/>
 			<PublicRoute
 				exact
-				restricted={false}
+				restricted={true}
```
Comment (svitlua) [inline client/src/containers/routing/index.tsx:51]: should it be restricted?

File: client/src/containers/profile/tabs/tasks/mocks.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ export const profileTasks = [
 		id: 'authored',
 		tasks: [
 			{
-				link: '/',
+				linkToAuthor: '/',
```
Comment (svitlua) [inline client/src/containers/profile/tabs/tasks/mocks.ts:43]: are you sure about this change?

**Review by svitlua — APPROVED** [review]

File: client/src/containers/routing/index.tsx (hunk 1 above)
Comment (sFLASHs) [reply client/src/containers/routing/index.tsx:51]: yes

File: client/src/containers/profile/tabs/tasks/mocks.ts (hunk 2 above)
Comment (sFLASHs) [reply client/src/containers/profile/tabs/tasks/mocks.ts:43]: yes

**Review by taras-dubyk — APPROVED** [review]

**Comment by sFLASHs** [comment]
[Unify application reducer state](https://trello.com/c/O0VqgBfH/59-unify-application-reducer-state)

**Pull request merged**